src/aind_metadata_service/sharepoint/client.py

```diff
@@ -292,37 +468,36 @@ def _get_filter_string(version: ListVersions, subject_id: str) -> str:
           A string to pass into a query filter
 
         """
-        default = (
+        default = ""
```

Comment:
We can probably remove the default string altogether if it's assumed a version needs to be passed into the function. You'll have to remove line 484 and change the if/elif starting on 485 to an if/else